client/signup/step-wrapper/index.jsx

```diff
@@ -65,26 +74,62 @@ export default React.createClass( {
 		}
 	},
 
+	recordComparePlansClick() {
+		analytics.ga.recordEvent( 'Upgrades', 'Clicked Compare Plans Link' );
+	},
+
+	renderComparePlans() {
+		const { stepName } = this.props;
+		if ( stepName !== 'plans' || ! isSkipPlansTestEnabled() ) {
+			return null;
+		}
+
+		const selectedSite = sites.getSelectedSite();
+		const url = plansLink( '/start/plans/compare', selectedSite );
+		const compareString = this.translate( 'Compare Plan Features' );
+
+		return (
+			<a href={ url } className="step-wrapper__compare-plans" onClick={ this.recordComparePlansClick }>
+				<Gridicon icon="clipboard" size={ 18 } />
+				{ compareString }
```

Comment:
Oh, do you mean how compareString is generated earlier and then added here? I suppose. This could have just used `this.translate()` inside the brackets. Truth be told, I probably would have done that, but I believe I copied this from somewhere else and then modified it for my own use. 😊